addons/languages/nias/pack/src/main/res/xml/nias_keyboards.xml

```diff
@@ -9,23 +9,5 @@
         index="1"
         layoutResId="@xml/nias_qwerty"
         nameResId="@string/nias_qwerty" />
-    <Keyboard
```

Comment:
Why are you removing these layouts?

Reply:
Those extra layouts were not used and have confused users. The standard one is now improved based on feedback I received. One keyboard layout with characters ö and ŵ is enough for Nias language.